src/accordion/accordion.js

```diff
@@ -78,6 +78,7 @@ angular.module('ui.bootstrap.accordion', ['ui.bootstrap.collapse'])
 
       scope.openClass = attrs.openClass || 'panel-open';
       scope.panelClass = attrs.panelClass || 'panel-default';
+      scope.customOptions = attrs.customOptions ? JSON.parse(attrs.customOptions) : '';
```

Comment:
use `angular.parseJson` instead of `JSON.parse` and set the default to `{}` not an empty string - that's just bad ju-ju.